extensions/vscode/src/multiStepInputs/newCredential.ts

```diff
@@ -246,8 +246,9 @@ export async function newCredential(
       step: thisStepNumber,
       totalSteps: state.totalSteps,
       value: currentName,
-      prompt: "Enter a Unique Nickname for your Credential.",
-      placeholder: "example: Posit Connect",
+      prompt:
+        "Enter a unique nickname for this server and API key combination.",
+      placeholder: "Posit Connect",
```

Comment:
Technically this is _just_ server — right? We still only accept/allow one key per server?

Reply:
you are correct @jonkeane . Perhaps we should change this to `Enter a unique nickname for this server`